matrix/spiral_print.py

```diff
@@ -7,6 +7,9 @@
 """
 
 
+from typing import List
```

Comment:
```suggestion
```
You can just use `list` to keep consistency with the file
This repository also targets at python 3.10+